File: Domains/Base/Workspace/CSharp/Domain/Base/Workspace/Data/InvokeResponse.cs
namespace Allors.Workspace.Data
{
    public class InvokeResponse : ErrorResponse
    {
        public bool hasErrors { get; set; }

        public string errorMessage { get; set; }

        public string[] versionErrors { get; set; }

        public string[] accessErrors { get; set; }

        public string[] missingErrors { get; set; }

        public PullResponseDerivationError[] derivationErrors { get; set; }
    }
}

File: Domains/Base/Workspace/Typescript/Domain/src/domain/base/data/responses/InvokeResponse.ts
import { ErrorResponse } from './ErrorResponse';
import { PullResponseDerivationError } from './PullResponseDerivationError';

export interface InvokeResponse extends ErrorResponse {
    hasErrors: boolean;
    errorMessage?: string;
    versionErrors?: string[];
    accessErrors?: string[];
    missingErrors?: string[];
    derivationErrors?: PullResponseDerivationError[];
}
